src/main/resources/net/rptools/maptool/language/i18n_uk.properties

```diff
@@ -2446,6 +2446,7 @@ tools.ignore_vbl_on_move.tooltip = Tokens will navigate around VBL if selected,
 tools.topology_mode_selection.vbl.tooltip = Draw Wall VBL.
 tools.topology_mode_selection.hill_vbl.tooltip = Draw Hill VBL.
 tools.topology_mode_selection.pit_vbl.tooltip = Draw Pit VBL.
+tools.topology_mode_selection.cover_vbl.tooltip = Draw Cover VBL.
```

Comment:
This should be placed in `i18n.properties` instead.

Reply:
You'd think I would've noticed that file was there.